connectors/sources/onedrive.py

```diff
@@ -206,18 +325,143 @@ def get_default_configuration(cls):
 
     async def close(self):
         """Closes unclosed client session"""
-        await self.get_client.close_session()
+        await self.client.close_session()
 
     async def ping(self):
         """Verify the connection with OneDrive"""
         try:
-            await anext(self.get_client.get(url_name=ENDPOINTS["PING"]))
+            url = parse.urljoin(BASE_URL, ENDPOINTS[PING])
+            await anext(self.client.get(url=url))
             self._logger.info("Successfully connected to OneDrive")
         except Exception:
             self._logger.exception("Error while connecting to OneDrive")
             raise
 
+    def _pre_checks_for_get_content(
+        self, attachment_extension, attachment_name, attachment_size
+    ):
+        if attachment_extension == "":
+            self._logger.debug(
+                f"Files without extension are not supported, skipping {attachment_name}."
+            )
+            return False
+
+        if attachment_extension.lower() not in TIKA_SUPPORTED_FILETYPES:
+            self._logger.debug(
```

Comment:
Missed this in the first review. If an oversized file is worth a warning log, so does unsupported extension (and file without an extension)